gtsfm/densify/patchmatchnet_data.py

```diff
@@ -112,6 +112,10 @@ def select_src_views_depth_ranges(self) -> Tuple[np.ndarray, np.ndarray]:
                 # Check if i1 is a valid image with estimated camera pose, then get its id for patchmatchnet
                 pm_i1 = self._camera_idx_to_patchmatchnet_idx.get(i1, -1)
 
+                # Calculate the depth only if i1 is an image with estimated pose
+                if pm_i1 < 0:
+                    continue
+
```

Comment:
can we refactor this check to first check if the key is present in the dictionary, and to return if not? It will be more readable.

Reply:
sure, good idea, i'll update it